src/number/n_GF.jl

```diff
@@ -125,9 +124,9 @@ end
 ###############################################################################
 
 function -(x::n_GF)
-    C = parent(x)
-    ptr = libSingular.n_Neg(x.ptr, C.ptr)
-    return C(ptr)
+   c = parent(x)
+   GC.@preserve x c p = libSingular.n_Neg(x.ptr, c.ptr)
```

Comment:
I think it's a bit more visually clear to write here and elsewhere the `@preserve` within the `=` expression, i.e. 
```suggestion
   p = GC.@preserve x c libSingular.n_Neg(x.ptr, c.ptr)
```
But it's not at all a suggestion to update this PR, just for future ones.

Reply:
noted.